lib/challenge.js

```diff
@@ -1,305 +1,327 @@
+ */
+
+function prompt3() {
+  for (let i = -5; i <= 5; i++) {
+    console.log(i);
+  }
+}
+
+// COMMENT IN THE BELOW LINE OF CODE TO TEST YOUR OUTPUT IN THE BROWSER!
+// prompt3()
+// 🌟 MAKE A COMMIT: "Complete prompt 3"
+// git add .
+// git commit -m "Complete Prompt 3"
+/*
+ * Prompt 4:
+ *
+ * Create a loop that counts down from 10 to 0, printing each number (including 10 and 0).
+ */
+
+function prompt4() {
+  for (let i = 10; i >= 0; i--) {
+    console.log(i);
+  }
+}
+
+// COMMENT IN THE BELOW LINE OF CODE TO TEST YOUR OUTPUT IN THE BROWSER!
+// prompt4()
+// 🌟 MAKE A COMMIT: "Complete prompt 4"
+// git add .
+/*git commit -m "Complete prompt 4"
+ * Prompt 5:
+ *
+ * Create a loop that counts down from 5 to -5, printing each number (including 5 and -5).
+ */
+function prompt5() {
+  for (let i = 5; i >= -5; i--) {
+    console.log(i);
+  }
+  
+}
+
+// COMMENT IN THE BELOW LINE OF CODE TO TEST YOUR OUTPUT IN THE BROWSER!
+// prompt5()
+// 🌟 MAKE A COMMIT: "Complete prompt 5"
+// git add.
+// git commit -m "complete prompt 5"
+/*
+ * Prompt 6:
+ *
+ * Create a loop that counts from 0 to 50 (inclusive) in multiples of 2 (instead of 1),
+ * printing each number.
+ */
+
+function prompt6() {
+  for (let i = 0; i <= 50; i+=2) {
+    console.log(i);
+  }
+}
+
+// COMMENT IN THE BELOW LINE OF CODE TO TEST YOUR OUTPUT IN THE BROWSER!
+// prompt6()
+// 🌟 MAKE A COMMIT: "Complete prompt 6"
+// git add .
+// git commit -m "complete prompt 6"
+/*
+ * Prompt 7:
+ *
+ * For the numbers 0 - 100 (inclusive), print out "I found a [ number ]. High five!" if the
+ * number is a multiple of five.
+ *
+ * Sample Output IN THE BROWSER:
+ *   - I found a 5. High five!
+ *   - I found a 10. High five!
+ */
+function prompt7() {
+  for (let i = 0; i <= 100; i++ ) {
+    if (i % 5 === 0) {
+      console.log("I have found a " + [i] + " High five!")
+    }
+  }
+}
+
+// COMMENT IN THE BELOW LINE OF CODE TO TEST YOUR OUTPUT IN THE BROWSER!
+prompt7()
+
+// 🌟 MAKE A COMMIT: "Complete prompt 7"
+
+/*
+ * Prompt 8:
+ *
+ * Create a variable called someNumber and assign it a random number between
+ * 0 and 100 (not inclusive). Hint: Use Math.random so that it generates a new number every time the code is run!
+ *
+ * Create a conditional that matches these requirements:
+ *   - if someNumber is less than 30, print "That's a small number!"
+ *   - if someNumber is between 30 and 60, print "The number is medium sized."
+ *   - if someNumber is greater than 60, print "We got a big one!"
+ */
+
+//  define someNumber here
+var someNumber = Math.floor(Math.random() * 99) +1;
+
+function prompt8() {
+  if (someNumber < 30) {
+  return "That's a small number!";
+} else if (someNumber >= 30 && someNumber <= 60) {
+  return "The number is medium sized.";
+      } else if (someNumber > 60) {
+        return "We got a big one!";
+  }
+  // print someNumber to the console in your function so you can see its value

```

Comment:
Be careful of the formatting.  To make things more readable I would rewrite this like:

```
if (someNumber < 30) {
  return "That's a small number!";
} else if (someNumber >= 30 && someNumber <= 60) {
  return "The number is medium sized.";
} else if (someNumber > 60) {
  return "We got a big one!";
}
  ```